tests/RangeCalendar.spec.jsx

```diff
@@ -200,6 +200,15 @@ describe('RangeCalendar', () => {
     expect(hoverValue[1].format(format)).toBe('2017-03-18');
   });
 
+  describe('calendar header', () => {
+    it('range date has selected, calendar header match date selected', () => {
+      const rangeCalendar = <RangeCalendar selectedValue={[moment('2000-09-03', format), moment('2000-10-28', format)]} timePicker={{}}/>;
+      const wrapper = mount(rangeCalendar);
+      expect(wrapper.find('.rc-calendar-month-select').at(0).text()).toEqual('Sep');
+      expect(wrapper.find('.rc-calendar-month-select').at(1).text()).toEqual('Oct');
+    });
+  });
```

Comment:
Seems this test could be same start & end date but different time. We should mock the click operation to open time panel and check if start & end are the same date.

Reply:
@zombieJ I have rewrite test case, please check